src/include/miopen/inline_vector.hpp

```diff
@@ -246,6 +347,27 @@ class InlineVector
     size_type real_size = 0;
 };
 
+template <typename T, std::size_t N>
+std::ostream& operator<<(std::ostream& os, const InlineVector<T, N>& iv)
+{
+    if(iv.empty())
+    {
+        os << "{}";
+    }
+    else
+    {
+        os << "{";
+        for(int i = 0; i < iv.size() - 1; i++)
```

Comment:
```suggestion
        for(size_t i = 0; i < iv.size() - 1; i++)
```
It emits `singned-unsigned comparison` warning.